test/css.js

```diff
@@ -137,10 +137,14 @@ describe('css', function() {
 
```

Comment:
Windows "randomly generated" class is not `1ezyc`. Update tests to:

- Check for the pattern `_index_[0-9a-z]+_1`
- Extract the found pattern to find it in the output css